views/secretWord.ejs

```diff
@@ -0,0 +1,10 @@
+<%- include("partials/head.ejs") %>
+<%- include("partials/header.ejs") %>
+  <p>The secret word is: <%= secretWord %></p>
+  <p>Would you like to change it?</p>
+  <form method="POST">
+    <label name="secretWord"></label>
+    <input name="secretWord" /><br />
+    <button>Submit</button>
+  </form>
+<%- include("partials/footer.ejs") %>
```

Comment:
Exactly this page looks really hard semantically. it can give let's say, a headache to style 
You could have had

```html
<HTML>
    <%- include("partials/head.ejs") %>
    <body>
         <%- include("partials/header.ejs") %>
         <main>
           <p>The secret word is: <%= secretWord %></p>
          <p>Would you like to change it?</p>
          <form method="POST">
            <label name="secretWord"></label>
            <input name="secretWord" /><br />
            <button>Submit</button>
            </form>
         </main>
        <%- include("partials/footer.ejs") %>
    </body>
</html>
```
> This make more sense semantically and help in user accessibility